scripts/globals/items.lua

```diff
@@ -1341,6 +1347,7 @@ xi.items =
     LAND_CRAB_MEAT                  = 4400,
     TORTILLA                        = 4408,
     HARD_BOILED_EGG                 = 4409,
+    MELON_JUICE                     = 4424,
```

Comment:
Can you please ensure that you're using the long name (`name`) for consistency (and tooling)?

For special characters, the following rules apply as well:
```
upName = upName.replace('+', 'P')
upName = upName.replace('_-', '_M')
upName = upName.replace('-', '_')
```